specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerAppsPatches.json

```diff
@@ -0,0 +1,566 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ContainerApps API Client",
+    "version": "2024-02-02-preview"
+  },
+  "schemes": [
+    "https"
+  ],
+  "host": "management.azure.com",
+  "produces": [
+    "application/json"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/patches": {
+      "get": {
+        "tags": [
+          "ContainerAppsPatches"
+        ],
+        "operationId": "ContainerAppsPatches_ListByContainerApp",
+        "description": "List ContainerAppsPatch resources by ContainerApp.",
```

Comment:
> ContainerAppsPatch resources by ContainerApp.", [](http://example.com/codeflow?start=28&length=48)

Can you give a better user-facing/problem-domain-related explanation of this? "Lists all the patches which are available to be manually or automatically applied to the given container app?"